server/app.js

```diff
@@ -19,12 +31,12 @@ app.use("/", indexRouter);
 app.use("/ping", pingRouter);
 
 // catch 404 and forward to error handler
-app.use(function(req, res, next) {
+app.use(function (req, res, next) {
   next(createError(404));
 });
 
 // error handler
-app.use(function(err, req, res, next) {
+app.use(function (err, req, res, next) {
```

Comment:
More of style choice than a need to add, but we could use ES6 arrow functions here. app.use((req, res, next) => {...})

Reply:
I agree, I think this was in the boilerplate and my linter just added the spaces.

Reply:
Figured 👍